datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeAdapterTest.java

```diff
@@ -67,6 +69,25 @@ void myafter() {
         simpleDatasafeService.destroyUser(userIDAuth);
     }
 
+    @ParameterizedTest
+    @MethodSource("minioOnly")
+    @SneakyThrows
+    void justCreateAndDeleteUserForMinioOnly(WithStorageProvider.StorageDescriptor descriptor) {
+        myinit(descriptor);
+        mystart();
+
+        // SimpleDatasafeAdapter does not use user profile json files, so only keystore and pubkeys should exist:
+        try (Stream<AbsoluteLocation<ResolvedResource>> stream = descriptor.getStorageService().get().list(BasePrivateResource.forAbsolutePrivate(descriptor.getLocation()))) {
+            assertThat(stream).extracting(it -> descriptor.getLocation().relativize(it.location()).asString())
+                    .extracting(string -> string.replaceAll("\\\\","/"))
```

Comment:
What is the purpose of replacing \\ with / after relativization?

Reply:
Hm, seems not really helpfull. yes.